src/AddTodoForm.jsx

```diff
@@ -1,32 +1,26 @@
-
-
 import React, { useState } from 'react';
 
-const AddTodoForm = ({ onAddTodo }) => {
-
-  const [todoTitle, setTodoTitle] = React.useState('');
-
-  // new Function
-  const handleAddTodo = (event) => {
-    // hint
-    event.preventDefault();
+function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = useState('');
 
-    const todoTitle = event.target.title.value;
-
-    console.log(todoTitle);
-
-    event.target.reset();
-
-    // last sentence of the part that says app,invoke the onAddTodo
-    onAddTodo(todoTitle);
+  const handleTitleChange = (event) => {
+    setTodoTitle(event.target.value);
+  };
 
+  const handleAddTodo = () => {
```

Comment:
Your submit function is not correctly integrated with a React application. The default behavior of submitting this form will conflict with your React app unless your prevent it, leading to some issues.

Reply:
`    event.preventDefault(); `